packages/core/upload/admin/src/components/AssetDialog/BrowseStep/index.js

```diff
@@ -135,6 +151,24 @@ export const BrowseStep = ({
 
             {(assetCount > 0 || folderCount > 0 || isSearching) && (
               <EndBlockActions pullRight>
+                <ActionButton paddingTop={1} paddingBottom={1}>
```

Comment:
The naming is a bit unfortunate, because it reads as if you are nesting `button` elements. Would `ActionWrapper` or `ActionContainer` be better?